cmd/root.go

```diff
@@ -18,5 +18,5 @@ var RootCmd *cmd.BeatsRootCmd
 
 func init() {
 	var runFlags = pflag.NewFlagSet(Name, pflag.ExitOnError)
-	RootCmd = cmd.GenRootCmdWithRunFlags(Name, "7.0.0-alpha1", beater.New, runFlags)
+	RootCmd = cmd.GenRootCmdWithRunFlags(Name, "", beater.New, runFlags)
```

Comment:
This uses a fallback to use the `default_version` of beats, which currently is `6.1.0`, so this is aligned with the other changes. However, why is the default version in master not set to `7.0.0-alpha` when everything else is?

Reply:
Master should be set to `7.0.0-alpha1` automatically that is why I removed https://github.com/elastic/apm-server/pull/132 here. Where in master is it not set to 7.0.0-alpha1?

Reply:
I have not had updated to this https://github.com/elastic/apm-server/pull/129/files#diff-942011c37b00c68bed7206f659059385.